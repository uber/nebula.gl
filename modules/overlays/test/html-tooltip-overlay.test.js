// @flow-ignore
import React from 'react';
import Renderer from 'react-test-renderer';

import DeckGL from '@deck.gl/react';
import HtmlTooltipOverlay from '../src/html-tooltip-overlay';

it('test HtmlTooltipOverlay no items shown', () => {
  const Component = (
    <DeckGL>
      <HtmlTooltipOverlay />
    </DeckGL>
  );
  const renderer = Renderer.create(Component);
  expect(renderer.toJSON()).toMatchSnapshot();
});
