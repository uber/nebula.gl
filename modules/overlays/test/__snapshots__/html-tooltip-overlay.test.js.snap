// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test HtmlTooltipOverlay no items shown 1`] = `
<div
  id="deckgl-wrapper"
  style={
    Object {
      "height": "100%",
      "left": 0,
      "position": "absolute",
      "top": 0,
      "width": "100%",
      "zIndex": 0,
    }
  }
>
  <canvas
    style={
      Object {
        "left": 0,
        "top": 0,
      }
    }
  />
</div>
`;
